### PR TITLE
Add Garmin Respiration

### PR DESCRIPTION
**Pull request by leite08**
Ticket: https://github.com/metriport/metriport-internal/issues/118

### Dependencies

- Upstream: https://github.com/metriport/metriport/pull/23
- Downstream: https://github.com/metriport/metriport/pull/25

### Description

- Add Garmin Respiration mapping

### Release Plan

- [ ] Release Upstream dependencies
- [ ] Update the SDK's version to the newest one
- [ ] Merge + release this

### CODE REVIEW

File: api/app/src/mappings/garmin/respiration.ts [hunk 1]
```diff
@@ -1,42 +1,82 @@
...
+    const uat = key;
+    const userData = values;
+    const user: User = {
+      userAccessToken: uat,
+    };
+    // flat list of user breaths/samples with date
+    const breaths = toBreaths(userData);
+    // now group those breaths by date and return as a Biometrics
+    const breathsByDate = groupBy(breaths, (i) => i.date);
+    // convert a
```
Comment (Goncharo) [inline api/app/src/mappings/garmin/respiration.ts:27]: typo?

File: api/app/src/mappings/garmin/respiration.ts [hunk 2]
```diff
@@ -1,42 +1,82 @@
...
+    const toUserBiometrics = (date: string): UserData<Biometrics> => {
+      const breathsOfDate = breathsByDate[date];
+      return {
+        user,
+        typedData: {
+          type,
+          data: {
+            metadata: { date, source: PROVIDER_GARMIN },
+            respiration: {
+              samples: breathsOfDate.map((v) => v.breath),
```
Comment (Goncharo) [inline api/app/src/mappings/garmin/respiration.ts:37]: See comment in https://github.com/metriport/metriport/pull/23 about the naming of `samples`

**Review by Goncharo — APPROVED** [review]
Minor comments - once addressed can 🚢

File: api/app/src/mappings/garmin/respiration.ts (hunk 1 above)
Comment (leite08) [reply api/app/src/mappings/garmin/respiration.ts]: yep, good catch

**Review by Orta21 — APPROVED** [review]

**Pull request merged**